unity/ComputerVision-UnityClient/Assets/Scripts/NetworkManager.cs

```diff
@@ -294,6 +318,7 @@ private double Deg2Rad(double deg)
             return (deg * Math.PI / 180.0);
         }
 
-    #endregion
+        #endregion
+        #endregion
```

Comment:
duplicate? Or if ending two regions, there may be an extra tab

Reply:
ending two regions, extra tab will be added, Thanks for the through review

Reply:
May want a // comment corresponding to a endregion.

Reply:
Correct, will do that from now